apps/base-docs/docs/pages/identity/smart-wallet/guides/linked-accounts/overview.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: 'Creating App Accounts with Coinbase Smart Wallet'
+description: 'Learn how to create and manage sub accounts using Coinbase Smart Wallet'
+---
+
+# Creating App Accounts with Coinbase Smart Wallet
+
+## Overview
+
+Coinbase Smart Wallet's self-custodial design requires a user passkey prompt for each wallet interaction that the user is presented with, such as a transaction or message signing. While this helps ensure the user is aware and required to approve of every interaction with their wallet, this also impacts user experience when using applications that may require frequent wallet interactions.
+
+To support using Coinbase Smart Wallet with user experiences that require more developer control over the wallet interactions, we've built Sub Accounts in conjunction with EIP-XXXX, a new wallet spec for creating hierarchical relationships between wallet accounts.
+
+Sub accounts allow you to provision wallet accounts that are directly embedded in your application for your users. You can control when a sub account is created for your user, and can interact with them just as you would with another wallet via the wallet provider, or other popular web3 libraries like OnchainKit, wagmi, viem, etc.
+
+These Sub accounts are linked to the end user's Coinbase Smart wallet through an onchain relationship. When combined with our [Spend Permission feature](/identity/smart-wallet/guides/spend-permissions/overview), this creates a powerful foundation for provisioning and funding app accounts, securely, while giving you, the developer, ample control over building the user experience that makes the most sense for your application.
+
+## What You'll Build
```

Comment:
I wonder if this guide could link to a repo with the complete example project in case folks wanted to just read along and not do all the copy-pasting required to get this into a working state?